api/v1alpha1/fenceagentsremediationtemplate_webhook.go

```diff
@@ -44,13 +44,19 @@ var _ webhook.Validator = &FenceAgentsRemediationTemplate{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (farTemplate *FenceAgentsRemediationTemplate) ValidateCreate() (admission.Warnings, error) {
 	webhookFARTemplateLog.Info("validate create", "name", farTemplate.Name)
-	return validateAgentName(farTemplate.Spec.Template.Spec.Agent)
+	if _, err := validateAgentName(farTemplate.Spec.Template.Spec.Agent); err != nil {
```

Comment:
when we extract both validations into a single validate function, use it here as well

Reply:
Fixed it. Changed it to the generic one.